eng/common/scripts/stress-testing/deploy-stress-tests.ps1

```diff
@@ -12,7 +12,9 @@ param(
     [switch]$Login,
 
     [Parameter(ParameterSetName = 'DoLogin')]
-    [string]$Subscription
+    [string]$Subscription,
+
+    [boolean]$CI = $false
```

Comment:
Did you want to default this similar to other places https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/TestResources/New-TestResources.ps1#L73.

Reply:
I just updated it to be a `switch` before I saw your comment. I'll add the default too.